devel/build_release_tarball.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+# Note: you might need to type your password a few times, once early, and once
+# at the end.
+#
+# This should be run on a release branch to make sure we can make a tarball and
+# at least build it on our workstations.  It will also copy the tarball into
+# ~/release (where the binaries usually go).
+#
+# Note that if this fails you may need to tweak the file list inside
+# devel/create_distro_tarball.sh
```

Comment:
You mean that this only runs on Ubuntu 14 right now, like the rest of our dev environment?